app/components/subcases/subcase-header.hbs

```diff
@@ -8,44 +8,48 @@
     <Auk::Toolbar::Group @position="right">
       {{#if (user-may "manage-agendaitems")}}
         <Auk::Toolbar::Item>
-          {{!-- template-lint-disable require-context-role --}}
-          <AuDropdown
-            data-test-subcase-header-actions-dropdown={{true}}
-            @skin="secondary"
-            @title={{t "actions"}}
-            @alignment="right"
-          >
-            {{#if this.canDelete}}
-              <AuButton
-                data-test-subcase-header-action-delete-subcase
-                @skin="link"
-                @alert={{true}}
-                {{on "click" (fn this.requestDeleteSubcase @subcase)}}
-                role="menuitem"
-              >
-                {{t "delete-subcase"}}
-              </AuButton>
-            {{/if}}
-            <AuButton
-              data-test-subcase-header-action-move-subcase
-              @skin="link"
-              {{on "click" this.triggerMoveSubcaseDialog}}
-              role="menuitem"
+          {{#if this.hasActions}}
+            {{!-- template-lint-disable require-context-role --}}
+            <AuDropdown
+              data-test-subcase-header-actions-dropdown={{true}}
+              @skin="secondary"
+              @title={{t "actions"}}
+              @alignment="right"
             >
-              {{t "move-subcase"}}
-            </AuButton>
-            {{#if this.canPropose}}
-              <AuButton
-                data-test-subcase-header-action-show-proposed-agendas
-                @skin="link"
-                {{on "click" (toggle "isAssigningToAgenda" this)}}
-                role="menuitem"
-              >
-                {{t "propose-for-agenda"}}
-              </AuButton>
-            {{/if}}
-          </AuDropdown>
-          {{!-- template-lint-enable require-context-role --}}
+              {{#if this.canDelete}}
+                <AuButton
+                  data-test-subcase-header-action-delete-subcase
+                  @skin="link"
+                  @alert={{true}}
+                  {{on "click" (fn this.requestDeleteSubcase @subcase)}}
+                  role="menuitem"
+                >
+                  {{t "delete-subcase"}}
+                </AuButton>
+              {{/if}}
+              {{#if @parliamentFlow}}
```

Comment:
Might be misunderstanding ticket, shouldn't this be `{{#if (not @parliamentFlow)}}` → only show "move subcase" action if (sub)case is not linked to a parliament flow?

Reply:
Whoops that was an `unless` I forgot to put back